tests/mocks.h

```diff
@@ -114,6 +109,7 @@ class MockBPFfeature : public BPFfeature
     has_d_path_ = std::make_optional<bool>(has_features);
     has_ktime_get_boot_ns_ = std::make_optional<bool>(has_features);
     has_kprobe_multi_ = std::make_optional<bool>(has_features);
+    has_uprobe_multi_ = std::make_optional<bool>(has_features);
```

Comment:
Not for this PR but it would be nice if setting these specific features can be more explicit in a particular test. `bpftrace->feature_ = std::make_unique<MockBPFfeature>(true);` doesn't tell me much about what is being enabled.

Reply:
Agreed, would be nice to have a way to simply setup `MockBPFfeature` with particular settings.